converter/src/main/java/gov/cms/qpp/conversion/correlation/model/Config.java

```diff
@@ -0,0 +1,55 @@
+package gov.cms.qpp.conversion.correlation.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder({
+	"decodeLabel",
+	"encodeLabels",
+	"goods"
+})
+public class Config {
```

Comment:
I guess I do not mind too much, but we don't need the `@JsonProperty("...")`.  The JSON will use the name of the fields by default.  And do we really care about the order of the JSON properties to specify `@JsonPropertyOrder(...)`?

This applies to the other classes in this `model` package too.

Reply:
Yeah, `@JsonProperty` should only be necessary if the exported property name needs to be different from the field name in the object.

Reply:
That was just what was generated automatically. I can remove that.